core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -100,6 +102,16 @@ oppia.directive('tutorCard', [
 
             AudioPlayerService.stop();
             AutogeneratedAudioPlayerService.cancel();
+
+            var _editorPreviewMode = (
```

Comment:
Let the player service handle all this stuff, including the event reporting.

Reply:
Done.